File: containsDuplicate.py
# Given an integer array nums, return true if any value appears at least twice in the array, and return false if every element is distinct

# Example:
# Input: nums = [1,2,3,1]
# Output: true

nums = [1,1,1,3,3,4,3,2,4,2]

valid = False

for num in nums:
    if num in nums[(nums.index(num) + 1):]:
        valid = True
        break
    else:
        continue
    
if valid:
    print(True)
else:
    print(False)
    

File: containsDuplicate.c
#include <stdbool.h>

bool containsDuplicate(int* nums, int numsSize) {
    int i, j;
    bool valid = false;

    for (i = 0; i < numsSize; i++) {
        for (j = i + 1; j < numsSize; j++) {
            if (nums[i] == nums[j]) {
                valid = true;
                break;
            }
        }
        if (valid) {
            break;
        }
    }

    return valid;
}
